### ISSUE TITLE
Correct remove button display validation set via `fileActionSettings`

### ISSUE THREAD

**Issue by keremburak** [kartik-v/bootstrap-fileinput#1390]
Hi,

I try to use fileinput plugin. When I use default settings, I cannot show "Remove" button whether I set the "showRemove : true" in "fileactionsettings" property but it doesnt shoe remove button icon.

My code block: in HTML:
  `<input type="file" id="myinput" />`

My code block: in Javascript:
```
<script>
    $(document).ready(function () {
        $("#myinput").fileinput({
            fileActionSettings: {
                showDrag: false,
                showZoom: true,
                showUpload: false,
                showRemove: false,
            },
        });
    });
</script>

```
What should I do in order to display "Remove file" button on image bottom-right

P.S: You can see it from the below picture.

![url](https://user-images.githubusercontent.com/908061/55834047-b168e200-5b21-11e9-8636-3ac89bfcd788.PNG)

**Comment by kelvinchoi** [comment]
I guess this is caused by the typo in the function called _renderFileActions while it's checking all the file action settings.  The parameter showDwn is used twice but showDel is missing.
```
if (!showUpl && !showDwn && !showDwn && !showZoom && !showDrag) {
    return '';
}
```

**Comment by kartik-v** [comment]
Read the docs on plugin usage and the usage modes. The remove button will only be enabled if you are using ajax mode when you have a valid uploadUrl](http://plugins.krajee.com/file-input/plugin-options#uploadUrl). 


For thumbnail content in initiaPreview you need to set [deleteUrl](http://plugins.krajee.com/file-input/plugin-options#deleteUrl)  or the *url* property within initialPreviewConfig to remove files.

**Comment by kartik-v** [comment]
Will need to check this.

**Comment by kartik-v** [comment]
Note that deletion of files that are not yet uploaded is only supported within ajax upload mode. Read about the [plugin usage modes](http://plugins.krajee.com/file-input#usage-modes). Which means you must set a valid [uploadUrl](http://plugins.krajee.com/file-input/plugin-options#uploadUrl) which will point to a server action that will process the upload via ajax.

So to summarize the remove button will be only shown in ajax mode for non uploaded file thumbnails.

On your other point (related to a code), I am noting a bug fix. But that should not be the reason for your issue.
